angular-ngrx-scss/src/app/state/profile/profile.state.ts

```diff
@@ -0,0 +1,164 @@
+import { UserState } from '../user';
+
+export interface ProfileState {
+  user?: UserState;
```

Comment:
I don't think user should be optional here - the only user we track is the one that's signed in, and they can't view any of this if they don't exist.

Reply:
I have it like this since the initial state in the reduce will have to fill in an empty user state. I don't know what the best practice is here, meaning is it better to start with an object with empty strings and zeros or just use null until it's defined.

Reply:
Oh good question - @ktrz any thoughts on best practice here?